geniza/corpus/management/commands/import_data.py

```diff
@@ -297,6 +298,68 @@ def add_document_language(self, doc, row):
             doc.language_note = "\n".join(notes_list)
             doc.save()
 
+    def parse_notes(self, doc, row):
+        if "PGPID" in row.notes:
+            old_pgpids = row.notes.split(": ")[-1]
+            doc.old_pgpids = [int(old_pgpid) for old_pgpid in old_pgpids.split(", ")]
+
```

Comment:
Can we split this into two separate functions for old pgpids and notes so that it can return the value (if any) to use when _creating_ the document, so that we don't have to update it again after the fact? It will help if we can avoid an additional save call, and it seems like it will be easier to test the logic too.

What I imagined is something like this:
```
doc = Document.objects.create(
                  id=row.pgpid or None,
                  old_pgpids=self.get_oldpgids(row.notes),
                  doctype=doctype,
                  description=row.description,
                  notes=self.get_notes(row)
              )
```

Reply:
Does this work for all cases? PGPID doesn't occur anywhere else in the notes as a non-old PGPID?

Reply:
No. All instances when "PGPID" is in the note2 column:
```
Old PGPID: 6160
Old PGPID: 6263
Old PGPID: 8843
India; Old PGPID: 9242
India; Old PGPID: 9261
India; Old PGPID: 9263
Old PGPID: 9563
Old PGPID: 11525
India; old PGPID: 6928
Old PGPID: 5711
India; old PGPID: 6580
India; old PGPID: 6589
India; Old PGPID: 9243
India; old PGPID: 16829
India; old PGPID: 9626
India; old PGPID: 5773
India; old PGPID: 30077
India; old PGPID: 6951
"Old PGPIDs: 9382, 16089"
Old PGPID: 7200
Old PGPID: 5861
Old PGPID: 5876
Old PGPID: 7988
India; Old PGPID: 8011
Old PGPID: 8037
Old PGPID: 7966
India; Old PGPID: 9207
Old PGPID: 7971
OId PGPID: 1727
Old PGPID: 7705
Old PGPID: 7627
Old PGPID: 7630
Old PGPID: 7631
Old PGPID: 7622
Old PGPID: 7731
Old PGPID: 7733
Old PGPID: 7742
India; Old PGPID: 7737
Old PGPID: 7738
India; Old PGPID: 8600
Old PGPID: 8631
India; old PGPID: 9636
India; old PGPID: 9416
old PGPID: 6948
```